semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -128,7 +126,10 @@ object Main {
           files
             .iterator()
             .asScala
-            .filter(f => Files.isRegularFile(f))
+            .filter { f =>
+              !isVisited(f) &&
+              Files.isRegularFile(f)
+            }
```

Comment:
Do we still need sorting? If not, let's go back to having all logic in `visitFile`.